server/routes/visit.test.ts

```diff
@@ -340,7 +340,7 @@ describe('/visit/:reference', () => {
           expect($('.govuk-back-link').length).toBe(0)
           expect($('[data-test="reference"]').text()).toBe('ab-cd-ef-gh')
 
-          expect(res.text).toContain(`This booking is not for ${supportedPrisons.BLI.replace('&', '&amp;')}`)
+          expect(res.text).toContain(`This booking is not for ${supportedPrisons.BLI.replace(/&/g, '&amp;')}`)
```

Comment:
Another issue flagged by CodeQL. I didn't realise that `replace` only replaces the first instance in a string. Using a regex with `/g` (global) option does them all. Obviously only a test file and not really an issue here but figured I may as well write it 'properly' 🤓!